### DISCUSSION TITLE
No SMTP servers were available for example.com. No hosts to try.

### DISCUSSION THREAD

**Discussion by krystalcode** [postalserver/postal#2935] [Help with using Postal]
- I have set up Postal and I can send messages.
- I have set up a domain and all DNS checks are reported as good by Postal.
- I have port 25 open via Docker.
- I can connect with telnet to port 25 and I can see in the docker logs that the connection reaches and is accepted by the smtp server container.

Still, whenever I try to send an email to the domain, either from another email provider or from within Postal, I get "No SMTP servers were available for example.com. No hosts to try.".

What am I missing for receiving emails?

**Comment by willpower232** [comment]
It might be that you have to change the internal DNS resolving setting in the config file because your particular installation requires it.

**Comment by jordantrizz** [comment]
I have the same, issue decided to post my own discussion https://github.com/orgs/postalserver/discussions/2986
